pages.pt_BR/common/bat.md

```diff
@@ -0,0 +1,28 @@
+# bat
+
+> Imprimir e concatenar arquivos.
+> Um clone de `cat` com destaque de sintaxe e integraço com Git. 
+
+- Imprimir os conteúdos de um arquivo para a standard output:
+
+`bat {{arquivo}}`
+
+- Concatenar vários arquivos para um arquivo de destino:
+
+`bat {{arquivo1}} {{arquivo2}} > {{arquivo_destino}}`
+
+- Concatenar vários arquivos para um arquivo de destino:
+
+`bat {{arquivo1}} {{arquivo2}} >> {{arquivo_destino}}`
```

Comment:
Acho que aqui temos o mesmo comando duplicado.

Reply:
Acho que não, na linha 12 tem >, na 16 tem >>

Reply:
Mas o funcionamento deles é o mesmo, certo? Ou estou enganado?

Reply:
Vou verificar. Se não for também vai ser preciso corrigir a página original

Reply:
Tem razão, é um erro! Vou corrigir neste!

Reply:
Se quiser, sinta-se a vontade para corrigir no principal também.

Reply:
Sim!